caluma/caluma_form/tests/test_history.py

```diff
@@ -372,3 +372,131 @@ def test_historical_table_answer(
     result = schema_executor(historical_query, variables=variables)
     assert not result.errors
     snapshot.assert_match(result.data)
+
+
+def test_history_question_answer(
```

Comment:
Can you add a comment on what aspect is being tested here?

Reply:
Gave it a better name instead

Reply:
I'd still like something along the lines of "test that historical question types retain the right type even when the question type for a given slug changes" or something similar. So you know what the intention is behind the test (I know I should do that more as well...)

Reply:
Did that